src/gba/memory.c

```diff
@@ -383,6 +385,8 @@ static void GBASetActiveRegion(struct ARMCore* cpu, uint32_t address) {
 		LOAD_32(value, address & (SIZE_CART0 - 4), memory->rom); \
 	} else if (memory->mirroring && (address & memory->romMask) < memory->romSize) { \
 		LOAD_32(value, address & memory->romMask, memory->rom); \
+	} else if (memory->vfame.cartType && address < BASE_CART1) { \
```

Comment:
I'm not sure why either of those checks are necessary? It'll always be less than BASE_CART_SRAM; that's what the switch is for, and it'll only ever be above BASE_CART1 if it's in one of the auxiliary cart regions, which are accessed identically, but have different wait states. Furthermore, memory->mirroring is really only for the Classic NES cartridge carts, so wouldn't be applicable here. The former condition makes this only apply if it's above the ROM bounds and a vfame cart, which is the real condition here, isn't it?